devices.js

```diff
@@ -8904,6 +8904,159 @@ const devices = [
+            await configureReporting.batteryVoltage(endpoint2);
+            await configureReporting.batteryPercentageRemaining(endpoint2);
+        },
+    },
+    {
+        zigbeeModel: ['888a434f3cfc47f29ec4a3a03e9fc442'],
+        model: 'ST21',
+        vendor: 'ORVIBO',
+        description: 'Temperature & humidity Sensor',
+        supports: 'temperature and humidity',
+        fromZigbee: [fz.temperature, fz.humidity, fz.battery_3V],
+        toZigbee: [],
+        meta: {configureKey: 2},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint1 = device.getEndpoint(1);
+            await bind(endpoint1, coordinatorEndpoint, ['msTemperatureMeasurement']);
+            const endpoint2 = device.getEndpoint(2);
+            await bind(endpoint2, coordinatorEndpoint, ['msRelativeHumidity', 'genPowerCfg']);
+            await configureReporting.temperature(endpoint1);
+            await configureReporting.humidity(endpoint2);
+            await configureReporting.batteryVoltage(endpoint2);
+            await configureReporting.batteryPercentageRemaining(endpoint2);
+        },
+    },
+    {
+        zigbeeModel: ['9f76c9f31b4c4a499e3aca0977ac4494'],
+        model: 'T30W3Z',
+        vendor: 'ORVIBO',
+        description: 'Smart light switch - 3 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off, fz.ignore_basic_report],
+        toZigbee: [tz.on_off],
+        meta: {configureKey: 1, multiEndpoint: true},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint1 = device.getEndpoint(1);
+            await bind(endpoint1, coordinatorEndpoint, ['genOnOff']);
+            const endpoint2 = device.getEndpoint(2);
+            await bind(endpoint2, coordinatorEndpoint, ['genOnOff']);
+            const endpoint3 = device.getEndpoint(3);
+            await bind(endpoint3, coordinatorEndpoint, ['genOnOff']);
+        },
+        endpoint: (device) => {
+            return {'top': 1, 'center': 2, 'bottom': 3};
+        },
+    },
+    {
+        zigbeeModel: ['074b3ffba5a045b7afd94c47079dd553'],
+        model: 'T21W2Z',
+        vendor: 'ORVIBO',
+        description: 'Smart light switch - 2 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off, fz.ignore_basic_report],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'top': 1, 'bottom': 2};
+        },
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+        },
+    },
+    {
+        zigbeeModel: ['095db3379e414477ba6c2f7e0c6aa026'],
+        model: 'T21W1Z',
+        vendor: 'ORVIBO',
+        description: 'Smart light switch - 1 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
```

Comment:
I also would expect a `configure` here like `T21W2Z` but only for endpoint 1

Reply:
Most of the 1 gang switches doesn't have `configure ` in `devices.js`. Anyway, should I add something like this in addition to above code?
```
meta: {configureKey: 1},
        configure: async (device, coordinatorEndpoint) => {
            const endpoint = device.getEndpoint(1);
            await bind(endpoint, coordinatorEndpoint, ['genOnOff']);
            await configureReporting.onOff(endpoint);
        },
```

Reply:
Yes, first test if you get a `state` update when togelling the switch via the button

Reply:
I've tested and received a `state `update
```
zigbee2mqtt:info  2020-05-15 18:12:15: MQTT publish: topic 'zigbee2mqtt/switch_one_gang', payload '{"state":"ON","linkquality":92}'
zigbee2mqtt:debug 2020-05-15 18:12:16: Received Zigbee message from 'switch_one_gang', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
zigbee2mqtt:info  2020-05-15 18:12:16: MQTT publish: topic 'zigbee2mqtt/switch_one_gang', payload '{"state":"OFF","linkquality":84}
```'

Reply:
Then it's not needed, thanks!

Reply:
I am receiving state update with and without the above mentioned code. So, I don’t know if it’s making any difference